enterprise/views.py

```diff
@@ -1624,6 +1627,16 @@ def post(self, request, enterprise_uuid, course_id):
         )
 
         # Create a link between the user and the enterprise customer if it does not already exist.
+        tracker_object = tracker.get_tracker()
+        data = {
+                'user_id': request.user.id,
+                'username': request.user.username,
+                'course_id': course_id,
+                'enterprise_customer_uuid': six.text_type(enterprise_customer.uuid)
+                }
+        tracker_object.enter_context('outer', data)
```

Comment:
Fields in "context" and "data" will be duplicated which is not seen in [other events generated by edx platform](https://github.com/edx/event-routing-backends/tree/master/event_routing_backends/processors/tests/fixtures/current). I'm just pointing this out so that it can be changed (if needed) before review of edX team.